src/components/search/HighlightRow.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+function HighlightRow({ booking, i, handleButtonClick }) {
```

Comment:
Align your code to use arrow function. See this link for the benefits of using arrow function. https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/